scanpy/external/tl/_harmony_timeseries.py

```diff
@@ -128,3 +131,4 @@ def harmony_timeseries(adata: AnnData, tp: str, n_components: Optional[int] = 10
     adata.obsp["harmony_aff"] = aff
     adata.obsp["harmony_aff_aug"] = aug_aff
     adata.uns["harmony_timepoint_var"] = tp
+    adata.uns["harmony_timepoint_connections"] = timepoint_connections
```

Comment:
This won’t work, anndata can’t write a dataframe with numerical column names to disk

Since the connections are currently tivial anyway, (just each one connected to the next), I decided to not store this.